src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -101,6 +101,30 @@ public class ReleaseService : IReleaseService
                 .OnSuccess(release => _mapper.Map<ReleaseViewModel>(release));
         }
 
+        public async Task<Either<ActionResult, List<ReleaseInternalNoteViewModel>>> GetInternalReleaseNotes(
```

Comment:
I think GetReleaseStatuses or GetReleaseStatusHistory would have been a better name for this as the note is just one part of the history that this overall change is now displaying.